views/beers.hbs

```diff
@@ -0,0 +1,11 @@
+<ul>
+    {{#each beersData}}
+    <li>
+        <h2>{{this.name}}</h2>
+        <h3>{{this.tagline}}</h3>
+        <img src="{{this.image_url}}">
+        <p>{{this.description}}</p>
```

Comment:
Give it a try using a partial. You can write it once and reuse it here and in the random beer as well.